### ISSUE TITLE
Deploying to GAE, seeing NeedIndexError: no matching index found.

### ISSUE THREAD

**Issue by robdodson** [GoogleChrome/big-rig#20]
I deployed my big-rig instance to GAE but found that after I add a project, as soon as I click on the project's name I get this error (giant stack trace below). Being a total GAE newb I *think* it's telling me to get datastore, which I enabled in the console, but I'm not sure where to give big-rig my API keys.

```
Traceback (most recent call last):
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 1535, in __call__
    rv = self.handle_exception(request, response, e)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 1529, in __call__
    rv = self.router.dispatch(request, response)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 1278, in default_dispatcher
    return route.handler_adapter(request, response)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 1102, in __call__
    return handler.dispatch()
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 572, in dispatch
    return self.handle_exception(e, self.app.debug)
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/webapp2-2.5.2/webapp2.py", line 570, in dispatch
    return method(*args, **kwargs)
  File "/base/data/home/apps/s~poly-rig/1.388738468218734710/handlers/project.py", line 238, in get
    self.response.write(template.render(data))
  File "/base/data/home/runtimes/python27/python27_lib/versions/third_party/jinja2-2.6/jinja2/environment.py", line 894, in render
    return self.environment.handle_exception(exc_info, True)
  File "/base/data/home/apps/s~poly-rig/1.388738468218734710/handlers/../templates/project/action.html", line 31, in top-level template code
    {% if actions.count() == 0 %}
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/ext/ndb/utils.py", line 142, in positional_wrapper
    return wrapped(*args, **kwds)
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/ext/ndb/query.py", line 1256, in count
    return self.count_async(limit, **q_options).get_result()
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/ext/ndb/tasklets.py", line 326, in get_result
    self.check_success()
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/ext/ndb/tasklets.py", line 369, in _help_tasklet_along
    value = gen.throw(exc.__class__, exc, tb)
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/ext/ndb/query.py", line 1299, in _count_async
    batch = yield rpc
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/ext/ndb/tasklets.py", line 455, in _on_rpc_completion
    result = rpc.get_result()
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/api/apiproxy_stub_map.py", line 613, in get_result
    return self.__get_result_hook(self)
  File "/base/data/home/runtimes/python27/python27_lib/versions/1/google/appengine/datastore/datastore_query.py", line 2922, in __query_result_hook
    yaml_index=yaml, xml_index=xml)
NeedIndexError: no matching index found.
The suggested index for this query is:
- kind: Action
  ancestor: yes
  properties:
  - name: name
```

**Comment by paullewis** [comment]
Yeah it could well be the case that GAE is building the indexes. Can you hit https://appengine.google.com/datastore/indexes and see if it's serving for all your indexes?

**Comment by robdodson** [comment]
This is what I see if I go there and check Datastore Indexes

<img width="794" alt="screen shot 2015-11-27 at 9 24 13 am" src="https://cloud.githubusercontent.com/assets/1066253/11444138/b40bbd3c-94e8-11e5-935f-a3d7e68073b7.png">

**Comment by paullewis** [comment]
_Should_ be fixed as of 54658e5eb7fa48b333737cad2150fbec370f95f2. Okay to close?

**Comment by robdodson** [comment]
Yup
